src/pages/admin/CustomerInstitutionAdminsForm.tsx

```diff
@@ -42,24 +44,24 @@ interface CustomerInstitutionAdminsFormProps {
 const CustomerInstitutionAdminsForm: FC<CustomerInstitutionAdminsFormProps> = ({ admins, customerInstitutionId }) => {
   const { t } = useTranslation('admin');
   const dispatch = useDispatch();
+  const [users, isLoadingUsers] = useFetchUsersForInstitution(customerInstitutionId);
   const [currentAdmins, setCurrentAdmins] = useState(admins);
+  const nonAdminUsers = users.filter((user) => !currentAdmins.some((admin) => admin.username === user.username));
```

Comment:
heftig logikk! blir litt svimmel egentlig. 

Er dette det samme?:
`!currentAdmins.some((admin) => admin.username === user.username))`
og
`currentAdmins.filter((admin) => admin.username !== user.username))`
?

Reply:
kan utvide funksjonen vi har i `role-helpers.ts` til å ta inn et array med roller vi vil filtrere på i stedet

Reply:
nei, det går heller ikke, siden vi her bare vil ha de som _ikke_ er Institution-admin... kan evt legge til et argument for `matching: boolean = true`, hvor `true` gir alle som har angitt rolle, og `false` gir alle som ikke har det

Reply:
høres lurt ut 👍

Reply:
fant en annen måte jeg tror er enklere, la det i egen PR inn til denne: https://github.com/BIBSYSDEV/NVA-Frontend/pull/915